File: Android-App/app/src/main/java/com/ite393group5/android_app/models/PersonalInfo.kt
package com.ite393group5.android_app.models

import com.ite393group5.android_app.utilities.DateSerializer
import kotlinx.serialization.Serializable
import java.time.LocalDate

@Serializable
data class PersonalInfo(
    val firstName: String? = null,
    val lastName:String? = null,
    val middleName:String? = null,
    val extensionName:String? = null,
    val gender:String? = null,
    val citizenship:String? = null,
    val religion:String? = null,
    val civilStatus:String? = null,
    val email:String? = null,
    val number:String? = null,
    @Serializable(with = DateSerializer::class)
    val birthDate: LocalDate? = null,
    val fatherName:String? = null,
    val motherName:String? = null,
    val spouseName:String? = null,
    val contactPersonNumber:String? = null,
)

File: web/app/models/PersonalInfo.php
<?php
namespace app\models;

class PersonalInfo
{
    public ?string $firstName;
    public ?string $lastName;
    public ?string $middleName;
    public ?string $extensionName;
    public ?string $gender;
    public ?string $citizenship;
    public ?string $religion;
    public ?string $civilStatus;
    public ?string $email;
    public ?string $number;
    public ?string $birthDate;
    public ?string $fatherName;
    public ?string $motherName;
    public ?string $spouseName;
    public ?string $contactPersonNumber;

    public function __construct($data)
    {
        $this->firstName = $data['firstName'] ?? null;
        $this->lastName = $data['lastName'] ?? null;
        $this->middleName = $data['middleName'] ?? null;
        $this->extensionName = $data['extensionName'] ?? null;
        $this->gender = $data['gender'] ?? null;
        $this->citizenship = $data['citizenship'] ?? null;
        $this->religion = $data['religion'] ?? null;
        $this->civilStatus = $data['civilStatus'] ?? null;
        $this->email = $data['email'] ?? null;
        $this->number = $data['number'] ?? null;
        $this->birthDate = $data['birthDate'] ?? null;
        $this->fatherName = $data['fatherName'] ?? null;
        $this->motherName = $data['motherName'] ?? null;
        $this->spouseName = $data['spouseName'] ?? null;
        $this->contactPersonNumber = $data['contactPersonNumber'] ?? null;
    }

    public function toArray()
    {
        return get_object_vars($this);
    }
}

?>
